invenio_data/base/templates/search/searchbar_frame.html

```diff
@@ -1,54 +0,0 @@
-{#
-# This file is part of CERN Analysis Preservation Framework.
-# Copyright (C) 2014, 2015 CERN.
-#
-# CERN Analysis Preservation Framework is free software; you can
-# redistribute it and/or modify it under the terms of the GNU General
-# Public License as published by the Free Software Foundation; either
-# version 2 of the License, or (at your option) any later version.
-#
-# CERN Analysis Preservation Framework is distributed in the hope that
-# it will be useful, but WITHOUT ANY WARRANTY; without even the
-# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
-# PURPOSE.  See the GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this software; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307,
-# USA.
-#}
-
-{% from "search/helpers.html" import collection_tree, portalbox_sidebar, search_also, search_form, search_form_javascript, record_brief_links with context %}
-{%- extends "search/searchbar_frame_base.html" -%}
-
-{% block body %}
-
-  {% if collection.id == 1 %}
-    <div style="text-align:center; margin-top:80px; margin-bottom:80px;font-style: bold; font-size: 400%;">
-      Data Analysis Preservation Demo
-    </div>
-  {% else %}
-    <h1>{{ collection.name_ln }}</h1>
-    <p>&nbsp;&nbsp;&nbsp;(submit <a href="/deposit/{{ collection.name|lower }}">new {{ collection.name }} analysis</a>)</p>
-  {% endif %}
-
-  {%- block search_form -%}
-    {{ search_form(collection, easy_search_form) }}
-  {%- endblock  %}
-
-  {% if collection.id == 1 %}
-    <div style="margin-top: 80px; text-align: center">
-      <a href="/collection/ALICE"><img src="/img/logo-alice.gif" width="80" border="0"></a>
-      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-      <a href="/collection/ATLAS"><img src="/img/logo-atlas.gif" width="80" border="0"></a>
-      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-      <a href="/collection/CMS"><img src="/img/cms-logo.gif" width="80" border="0"></a>
-      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-      <a href="/collection/LHCb"><img src="/img/lhcb-logo.jpg" width="100" border="0"></a>
-    </div>
-  {% endif %}
```

Comment:
Removing `searchbar_frame.html` would modify the look of the home page, e.g. the logos of the experiments would be gone.  We need to preserve it.